app/src/main/java/com/kodeco/android/countryinfo/MainActivity.kt

```diff
@@ -35,8 +39,13 @@ class MainActivity : ComponentActivity() {
                     CountryErrorScreen(
                         headline = "No Internet Connection",
                         subtitle = "Please Connect To Network"
-                    )
-                }
+                    ){
+                        val intent = Intent(this, MainActivity::class.java)
+                        finish()
+                        startActivity(intent);
+                    }
+                    }
+
                 }
             }
         }
```

Comment:
nitpick: These should be indented back 4 spaces for better readability:

```suggestion
                }

            }
        }
    }
```